mux/src/tmux_commands.rs

```diff
@@ -308,12 +308,14 @@ impl TmuxCommand for CapturePane {
 
         let unescaped = termwiz::tmux_cc::unvis(&result.output).context("unescape pane content")?;
         let unescaped = unescaped.replace("\n", "\r\n");
+        // capturep contents returned from guarded lines which always contain a tailing '\n'
+        let trim_unescaped = &unescaped[0..unescaped.len() - 1];
```

Comment:
I think this should probably happen before the `\n` is rewritten into `\r\n` so that we don't leave a stray `\r` on the end of this string?

Also, it would be good to make this safer if we end up with unescaped as an empty string; `unescaped.len().saturating_sub(1)` won't wrap to a really large number in that case.